src/project/qml/MuseScore/Project/internal/NewScore/ChooseInstrumentsAndTemplatesPage.qml

```diff
@@ -27,10 +27,11 @@ import Muse.Ui 1.0
 import Muse.UiComponents 1.0
 import MuseScore.Project 1.0
 import MuseScore.InstrumentsScene 1.0
-
 Item {
     id: root
 
+    required property InstrumentsOnScoreListModel instrumentsOnScoreModel
```

Comment:
This property should not be here. In this file, the `InstrumentsOnScoreListModel` is instantiated, so there should be no property. Properties are usually only used to pass information from outside the file into the file, or to expose information from inside the file to outside the file.

Reply:
Ok, got it.